Wind.elm

```diff
@@ -27,39 +30,52 @@ updateWind = updateWindSpeed << updateWindDirection
 updateWindSpeed : Model -> Model
 updateWindSpeed m =
   let (prob, seed') = generate (float 0 1) m.seed
-      windSpeed' = m.windSpeed + prob
+      windSpeed' = (m.windSpeed * 0.9) + (prob * 0.1)
```

Comment:
This smells of code duplication! Something like:

~~~ Haskell
slowlyChangeBy x d = (x * 0.9) + (d * 0.1) 
~~~